packages/frontend-shared/package.json

```diff
@@ -20,6 +20,7 @@
   "devDependencies": {
     "@antfu/utils": "^0.3.0",
     "@cypress-design/css": "^0.5.2",
+    "@floating-ui/dom": "1.0.1",
```

Comment:
why is this added to this package vs the app package?

Reply:
It's my mistake and I'll fix this my `Highlight` PR.